src/ontology/mondo-edit.obo

```diff
@@ -191216,7 +191184,7 @@ xref: OMIM:265400 {source="MONDO:equivalentTo"}
 xref: Orphanet:422 {source="OMIM:265400"}
 xref: UMLS:C1849552 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:265400"}
 is_a: MONDO:0001999 {source="MONDO:Redundant", source="OMIM:265400"} ! primary pulmonary hypertension
-is_a: MONDO:0008347 {source="Orphanet:422/btnt"} ! idiopathic and/or familial pulmonary arterial hypertension
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
do you meant the relationship? This is one of the QC checks- to check if there is a single child - parent relationship (ie a parent only has one child). We added this QC check to watch out for these because they _could_ signal something is wrong (and it's a good check because sometimes there is something wrong) but we can override it. I'm overriding it here.